website/integrations/services/fortigate-ssl/index.md

```diff
@@ -0,0 +1,239 @@
+---
+title: FortiGate SSLVPN
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## FortiGate SSLVPN
+
+> FortiGate is a firewall from FortiNet. It is a NGFW with layer7 inspection and able to become a part of a FortiNet security fabric.
+> -- https://www.fortinet.com/products/next-generation-firewall
+>
+> This guide explains how to setup a FortiGate to use authentik as SAML provider for SSLVPN authentication. It does not cover how to setup SAML for admin logins, that is a different configuration. If you need to setup SAML for admin logins see the FortiGate admin guide.
```

Comment:
```suggestion
> This guide explains how to setup a FortiGate to use authentik with a SAML provider for SSLVPN authentication. It does not cover how to setup SAML for admin logins, that is a different configuration. If you need to setup SAML for admin logins see the FortiGate admin guide.
```

Reply:
@nicedevil007 @DragonWin by "Fortigate admin guide" do you mean the FortiGate official docs, or the other file in the PR? Let's add a link to the other file, if that is what you mean.

Reply:
It should have been a reference to the guide done by @nicedevil007 (admin guide), the documentation from FortiNet focus mostly on Azure integration and integration with their own products, which are not applicable to setting up SAML with authentik.